modules/claims_api/app/controllers/concerns/claims_api/poa_verification.rb

```diff
@@ -32,6 +32,8 @@ def valid_poa_code?(poa_code)
       #
       # @raise [Common::Exceptions::Forbidden] if @current_user is not a representative
       def validate_user_is_accredited!
+        return nil if @current_user.first_name.nil? || @current_user.last_name.nil?
```

Comment:
returning `nil` here will allow the `validate_user_is_accredited!` check to pass in a scenario where we can't look up the rep. This should either raise an error or you can remove this change since the `for_user` method will now return `nil` and hit `raise ::Common::Exceptions::Forbidden if representative.blank?`